src/DB_Command.php

```
@@ -2393,4 +2434,319 @@ protected function apply_sql_mode_compat_init_command( &$mysql_args, $assoc_args
+
+		if ( ! defined( 'WP_CONTENT_DIR' ) ) {
+			define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
+		}
+
+		$wpdb_file = file_exists( ABSPATH . WPINC . '/class-wpdb.php' )
+			? ABSPATH . WPINC . '/class-wpdb.php'
+			: ABSPATH . WPINC . '/wp-db.php';
+
+		// Load prerequisite WordPress files if not already loaded.
+		$required_files = [
+			ABSPATH . WPINC . '/load.php',
```

PHPStan (2.2.5)

[error] Constant ABSPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)